src/sonic-config-engine/minigraph.py

```diff
@@ -1679,7 +1679,8 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         port_default_speed =  port_speeds_default.get(port_name, None)
         port_png_speed = port_speed_png[port_name]
 
-        if switch_type == 'voq':
+        # Add a check for for voq, T1
+        if switch_type in ['voq', 'LeafRouter']:
```

Comment:
switch_type will not be 'leafRouter'  you can use this check `results['DEVICE_METADATA']['localhost']['type'].lower() == 'leafrouter'` to check of the device is T1

Reply:
fixed